elementary/monitor/api/tests/tests.py

```diff
@@ -433,19 +459,25 @@ def _get_total_tests_results(
     ) -> Dict[str, TotalsSchema]:
         totals: Dict[str, TotalsSchema] = dict()
         for test in test_metadatas:
+            if not test.model_unique_id:
+                continue
+
             self._update_test_results_totals(
                 totals_dict=totals,
                 model_unique_id=test.model_unique_id,
-                status=test.latest_run_status,
+                status=test.latest_run_status or "unknown",
```

Comment:
How can `latest_run_status` be `None`?
Where does this `status` go to?

Reading the code seems like it won't do anything:
```py
    def add_total(self, status):
        total_adders = {
            "error": self._add_error,
            "warn": self._add_warning,
            "fail": self._add_failure,
            "pass": self._add_passed,
        }
        adder = total_adders.get(status)
        if adder:
            adder()
```